Casks/pdk.rb

```diff
@@ -21,5 +21,11 @@
   name 'Puppet Development Kit'
   homepage 'https://github.com/puppetlabs/pdk'
 
+  pdk_bins = '/opt/puppetlabs/pdk/bin'
```

Comment:
These files are regenerated from templates. Can you move this text to `templates/pdk.rb.erb` and `templates/puppet-bolt.rb.erb`?